docs/0_0_basics.fsx

```diff
@@ -47,6 +47,7 @@ The central type that gets created by all Chart constructors is `GenericChart`,
 *)
 
 (***do-not-eval***)
+[<NoComparison>]
```

Comment:
It yielded a warning that there are types which do not override Equals. So it suggested me adding this attribute.

Reply:
I think here we can safely ignore that, because this is just a showcase of how the type looks like internally. But ill rework this page anyways so it can stay as it is